Python/Product/PythonTools/PythonTools/Intellisense/PythonSuggestedActionsSource.cs

```diff
@@ -60,16 +61,20 @@ ITextBuffer textBuffer
         }
 
         public async Task<bool> HasSuggestedActionsAsync(ISuggestedActionCategorySet requestedActionCategories, SnapshotSpan range, CancellationToken cancellationToken) {
-            var textBuffer = _textBuffer;
             var pos = _view.Caret.Position.BufferPosition;
-            pos = _view.BufferGraph.MapDownToFirstMatch(pos, PointTrackingMode.Positive, EditorExtensions.IsPythonContent, PositionAffinity.Successor) ?? pos;
-            var line = pos.GetContainingLine();
-            if (pos.Position < line.End.Position) {
+            if (pos.Position < pos.Snapshot.Length - 1) {

```

Comment:
Fairly sure this will behave strangely at the end of lines now, which is why I originally changed this to look at `line.End`